### DISCUSSION TITLE
导入到 Craft

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#3144] [插件说明]
描述
---

> 将阅读模式的内容导入到 Craft。

https://user-images.githubusercontent.com/81074/144559583-25335b63-9727-4413-8fe8-4490c148fc7a.mp4

地址
---

[https://simpread.ksria.cn/plugins/details/Jv9XKxcUBE](https://simpread.ksria.cn/plugins/details/Jv9XKxcUBE)

快捷键
---

> 支持全局快捷键 `c a`

选项
---

![image](https://user-images.githubusercontent.com/81074/144560417-3ac0a35a-1660-4260-b5fd-071821264609.png)

如何设置
---

### 获取 Space ID

打开 Craft 然后找一篇 Document，进入后根据下图所示选择即可。

![image](https://user-images.githubusercontent.com/81074/144559357-d8e2dc95-e851-4368-b2a8-a2f2fd9e4ceb.png)

### 设置 Space ID

打开任意文本编辑器，粘贴上步得到的 URL，会得到类似下图的内容。

`craftdocs://open?blockId=******-60b9-6efb-b818-d2538e2c7dc6&spaceId=******-4de1-7e31-b22d-******`

复制 `spaceId` 后面的值粘贴到下图即可。

![image](https://user-images.githubusercontent.com/81074/144558808-cbde432a-754e-4943-b209-f66fa41b6546.png)

### 设置 Folder ID，留空即可。

更新日志
---

### 1.0.0

- 设置 Space ID / Folder ID

- 保存

注意
---

> 根据 [Craft 文档描述](https://support.craft.do/hc/en-us/articles/360020168838-Using-URL-Scheme)，通过此方式导入的内容，仍是外链图片形式。

![image](https://user-images.githubusercontent.com/81074/144566795-819e5bab-3b4c-4ae3-af85-7650da8fe4cd.png)

> 如果需要确保导入的图片为 Craft 图床方案的话，请通过 [简悦导出 Textbundle](http://ksria.com/simpread/docs/#/Textbundle) 后再上传到 Craft 的方式。

![image](https://user-images.githubusercontent.com/81074/144566960-737cfedc-9a64-42c9-aa52-d9b5f26fb57d.png)

> 导入 Textbunle 后 Craft 会自动将这些图片上传到它自己的图床。

![image](https://user-images.githubusercontent.com/81074/144567614-08816445-9789-495a-a99f-410c5a1765ce.png)

以上部分内容来自 [简悦社区](https://t.me/simpreadgroup) 用户 Larry 的说明。

引申
---

> 安装插件或带有插件触发器的插件失效的 [解决方案](https://github.com/Kenshin/simpread/discussions/2342)。

**Comment by alivelee** [comment]
简悦导出 Textbundle，这个链接404

**Reply by Kenshin** [reply]
3ks, Changed  Done

**Comment by JimmyLv** [comment]
跟进速度太快啦！点赞 o(￣▽￣)ｄ
